go/internal/feast/model/basefeatureview.go

```diff
@@ -56,3 +57,11 @@ func (fv *BaseFeatureView) ProjectWithFeatures(featureNames []string) *FeatureVi
 		Features: features,
 	}
 }
+
+func CreateBaseFeatureView(name string, features []*Feature, projection *FeatureViewProjection) *BaseFeatureView {
```

Comment:
if this is for tests only, it should in test package

Reply:
Done.